pl_bolts/datamodules/sklearn_datamodule.py

```diff
@@ -6,14 +7,10 @@
 from pytorch_lightning import LightningDataModule
 from torch.utils.data import Dataset, DataLoader
 
-try:
+
+_SKLEARN_AVAILABLE = find_spec("sklearn") is not None
+if _SKLEARN_AVAILABLE:
```

Comment:
What happened to the warning if not found?

Reply:
Good catch! In #346, I'm suggesting a possibly better way to import optional packages. Would be wonderful if you leave your thoughts there as well :]